scripts/apps/authoring/authoring/directives/WordCount.js

```diff
@@ -1,14 +1,21 @@
 import * as helpers from 'apps/authoring/authoring/helpers';
 
-WordCount.$inject = ['gettextCatalog'];
-export function WordCount(gettextCatalog) {
+/**
+ * @ngdoc directive
+ * @module superdesk.apps.authoring
+ * @name sdWordCount
+ * @description
```

Comment:
We usually put the description all on one line. It doesn't bother me too much but it'd be nice for consistency.